src/tests/unit/packages/bpmn/bpmn-pool/bpmn-transaction-component-test.tsx

```diff
@@ -0,0 +1,22 @@
+import { wrappedRender } from '../../../test-utils/render';
+import * as React from 'react';
+import { CSSProperties } from 'react';
+import { Multiline } from '../../../../../main/utils/svg/multiline';
+import { BPMNPool } from '../../../../../main/packages/bpmn/bpmn-pool/bpmn-pool';
+import { BPMNPoolComponent } from '../../../../../main/packages/bpmn/bpmn-pool/bpmn-pool-component';
+
+// override getStringWidth, because it uses by jsdom unsupported SVGElement methods
```

Comment:
@matthiaslehnertum I don't need this locally to get the tests passing, perhaps we need to pin a later version of JSDOM to resolve this issue? Additionally, this method of patching could potentially break subsequent tests as it is not reversed, I suspect in these cases polyfilling DOM APIs will yield a more robust and explicit work-around.